settings.py

```diff
@@ -208,8 +210,22 @@
     if delivered_lang in PROD_LANGUAGES:
         LANGUAGE_URL_MAP[requested_lang.lower()] = delivered_lang
 
-# Override Django's built-in with our native names
+
+def get_locales():
+    locales = {}
+    file = os.path.join(ROOT, 'kuma', 'languages.json')
+    json_locales = json.load(open(file, 'r'))
+    for locale, meta in json_locales.items():
+        locales[locale] = _Language(meta['english'],
+                                    meta['native'],
+                                    meta['iso639_1'])
```

Comment:
Generally curious why this needs to be a namedtuple and not just the dict imported from the json file?